fastly/waf_rules.go

```diff
@@ -49,6 +49,8 @@ type ListWAFRulesInput struct {
 	FilterTagNames []string
 	// Limit the returned rules to a set by publishers.
 	FilterPublishers []string
+	// Limit the returned rules to a set by modsecurity rule IDs.
+	FilterModSecIDs []int
 	// Excludes individual rules by modsecurity rule IDs.
 	ExcludeMocSecIDs []int
```

Comment:
Out if interest this looks like a typo... right? Should be `ExcludeModSecIDs`? If so we should add a TODO to update and bump to a new major version.